app/views/templates/search_bar_template.scala.html

```diff
@@ -27,7 +27,7 @@
 
     <div class="search-field search-field-darker">
         <div class="search-input search-input-with-button">
-            <label for="term" class="@if(!showLabel){visuallyhidden} bold">@messages("softwareChoices.search.button.text")</label>
+            <h2 class="@if(!showLabel){visuallyhidden} search-label">@messages("softwareChoices.search.button.text")</h2>
```

Comment:
You still need the label element as before, it just needs to be _wrapped_ in a h2

e.g.

h2
- label for=x
input id=x